src/components/global/ContactFormModal.jsx

```diff
@@ -70,6 +76,7 @@ const ContactFormModal = ({ showContactFormModal, setShowContactFormModal }) =>
             variant="standard"
```

Comment:
<DialogContentText>
            For any questions or to just reach out, contact us today!
          </DialogContentText>
          
          this can be entirely removed IMO

Reply:
Should remove the import on line 10 in that case.

I did it in the other open PR.

Reply:
I removed it with the push I made.